### DISCUSSION TITLE
Observing Convergence in a Running Scenario

### DISCUSSION THREAD

**Discussion by tyw-0628** [Kidel/Kathara#235] [Q&A] [answered]
Hi all. My aim is to observe the convergence of the network so I decided to stop/delete a connection/device in the running Kathara network by `kathara vclean -n deviceName'` but I can still `kathara connect` to the device and successfully ping the device. If I want to delete a link between 2 devices or stop a device as mentioned without re-running the scenario, what should I do?

**Comment by tcaiazzi** [comment]
Hello @tyw-0628, 

This seems strange.  Did you use the `kathara lstart` command to run the network scenario? If yes, you have to use `kathara lclean <device_name>` to stop a running device inside a network scenario. 

Let me know, if it helps! Otherwise, can you please share the lab and all the commands you used to reproduce the error? 

Many thanks, 
Tommaso

**Reply by tyw-0628** [reply]
Hi @tcaiazzi , 

Thanks for your reply. I am sure I have `kathara lclean` before `kathara lstart` or even `kathara lrestart`. Here comes the situation. 
![1B7C994B-4E34-4EA9-9448-4D2FAB558872](https://github.com/KatharaFramework/Kathara/assets/81566257/e1c8ec63-106c-4881-af18-f8175662d01b)

Here is the Kathara folder which is very simple with 2 clients connected to a router. 
[lab.zip](https://github.com/KatharaFramework/Kathara/files/12250132/lab.zip)

Btw, is there any way to temporarily remove a connection (collision domain) in command line?

**Reply by tcaiazzi** [reply, marked as answer]
HI @tyw-0628,

To delete a running device inside a scenario you have to use the `kathara lclean` command. 

Try to use the following command inside the lab directory:
```
kathara lclean router0
``` 

The `kathara vclean` command only delete devices deployed using the `kathara vstart` command. 

As a general rule, to interact with devices deployed with the `kathara lstart` command, you have to use "l-prefixed commands".

For a more complete guide on the Kathará commands, you can check the [man pages](https://www.kathara.org/man-pages/kathara.1.html). 

Let me know if this resolve your problem and don't hesitate to ask if you need further help!

**Reply by tyw-0628** [reply]
Hi @tcaiazzi,

Thanks for the clarification. I will check the manual for more detail. 

BIG THANKS!

(To update, as my version of Kathara is 3.3.0, I am not able to remove any collision domain in command line.)
